src/researches/getPassiveVoice.js

```diff
@@ -1,110 +1,122 @@
-var getSentences = require( "../stringProcessing/getSentences.js" );
-var stripHTMLTags = require( "../stringProcessing/stripHTMLTags.js" ).stripFullTags;
-var getLanguage = require( "../helpers/getLanguage.js" );
-var Sentence = require( "../values/Sentence.js" );
+const getSentences = require( "../stringProcessing/getSentences.js" );
+const stripHTMLTags = require( "../stringProcessing/stripHTMLTags.js" ).stripFullTags;
+const getLanguage = require( "../helpers/getLanguage.js" );
+const Sentence = require( "../values/Sentence.js" );
 
-// Imports used for English, French and Spanish.
-var getSentencePartsDefault = require( "./passiveVoice/getSentenceParts.js" );
-var determinePassivesDefault = require( "./passiveVoice/determinePassives" );
+const forEach = require( "lodash/forEach" );
 
-// Imports used for German.
-var getSentencePartsGerman = require( "./german/passiveVoice/getSentenceParts.js" );
-var determinePassivesGerman = require( "./german/passiveVoice/determinePassives.js" );
+const determinePassiveSentencePart = require( "./passiveVoice/periphrastic/determinePassiveSentencePart.js" );
+const determinePassiveSentence = require( "./passiveVoice/morphological/determinePassiveSentence.js" );
 
-var forEach = require( "lodash/forEach" );
+const getSentencePartsDefault = require( "./passiveVoice/periphrastic/getSentenceParts.js" );
```

Comment:
Renamed to getPeriphrasticSentencePartsDefault and get PeriphrasticSentencePartsGerman respectively. It's somewhat excessive, in my opinion, since SentencePart is only relevant for periphrasic passive voice marking, but if you think it helps reading the code, I do not mind this change.